### PR TITLE
Update README & Add compositors list

### PR DESCRIPTION
**Pull request by PolyMeilex**
[rendered](https://github.com/Smithay/smithay/blob/217d6c87b530a43b8fcf8859eb0cfa2541c8ae93/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,44 +31,31 @@ Also:
 
 ## Anvil
 
-### Dependencies
+Like others, Smithay as a compositor library has its own sample compositor: anvil.
```
Comment (i509VCB) [inline README.md:34]: Like what?

File: anvil/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Anvil
+
+Like others, Smithay as a compositor library has its own sample compositor: anvil.
```
Comment (i509VCB) [inline anvil/README.md:3]: Ditto

File: README.md (hunk 1 above)
Comment (Drakulix) [reply README.md:34]: libweston? :sweat_smile:

Comment (PolyMeilex) [reply README.md:34]: wlroots, with tinywl?

**Comment by wiki-me** [comment]
What about [Anodium](https://github.com/PolyMeilex/Anodium) and [aerugo](https://github.com/i509VCB/aerugo), shouldn't they also be added?

**Comment by PolyMeilex** [comment]
> What about [Anodium](https://github.com/PolyMeilex/Anodium) and [aerugo](https://github.com/i509VCB/aerugo), shouldn't they also be added?

- i509VCB has stated on matrix that aerugo should not be on the list yet.
- As for Anodium, as you can see from the commit history I borderline abandoned it, so it would be weird to list it.

File: anvil/README.md (hunk 2 above)
Comment (PolyMeilex) [reply anvil/README.md]: Replaced with proper description

**Comment by wiki-me** [comment]
> As for Anodium, as you can see from the commit history I borderline abandoned it, so it would be weird to list it.

Projects sometimes have periods when they are inactive , if you are working on infrastructure for it and planning to return to it (and even better if you will mention it on the read me), I don't think that is a problem.

Also shoudn't the list be on the website? (and link from the read me to the markdown page that will generate the HTML)

Or maybe upgrade this to a "awesome-smithay" repo and link to it from the read me (like lemmy [does](https://github.com/LemmyNet/lemmy/blob/main/README.md)).

File: README.md [hunk 3]
```diff
@@ -31,44 +31,31 @@ Also:
...
-
-- `--x11`: start anvil as an X11 client. This allows you to run the compositor inside an X11 session or any compositor supporting XWayland. Should be preferred over the winit backend where possible.
-- `--winit`: start anvil as a [Winit](https://github.com/tomaka/winit) application. This allows you to run it
-  inside of an other X11 or Wayland session.
-- `--tty-udev`: start anvil in a tty with udev support. This is the "traditional" launch of a Wayland
-  compositor. Note that this requires you to start anvil as root if your system does not have logind
-  available.
+- `xwayland`
 
 ## Contact us
```
Comment (elinorbgr) [inline README.md:61]: Maybe we can remove the mention of the IRC chatroom while we're at it (just below here)

Comment (PolyMeilex) [reply README.md:61]: Done

File: README.md (hunk 1 above)
Comment (PolyMeilex) [reply README.md]: Done

**Comment by i509VCB** [comment]
Anything else blocking this?

**Review by elinorbgr — APPROVED** [review]

**Pull request merged**